### ISSUE TITLE
9.1.4.11/4.2 Media files marked as favorites using stars - Depending on the background, the stars do not meet the contrast requirements for graphical informative elements, as the contrast ratio with the background is below 3:1. It is recommended to place these elements outside of images.

### ISSUE THREAD

**Issue by szaimen** [nextcloud/server#42217]
https://report.bitvtest.de/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-3dbfb7615a-v4-n2

9.1.4.11/4.2 Media files marked as favorites using stars - Depending on the background, the stars do not meet the contrast requirements for graphical informative elements, as the contrast ratio with the background is below 3:1. It is recommended to place these elements outside of images.

![image](https://github.com/nextcloud/server/assets/42591237/342c72a8-80d2-4f62-897c-b05436d49760)

**Comment by emoral435** [comment]
For this case, can't we just apply a black / contrasting image background color behind the star? That would be a solution that can work

**Comment by szaimen** [comment]
I think this one was already solved with https://github.com/nextcloud/photos/issues/2159 but I couldnt find it in our board somehoe, thus I recreated it.
